pkgs/development/interpreters/wasmer/default.nix

```diff
@@ -2,44 +2,62 @@
 , lib
 , rustPlatform
 , fetchFromGitHub
-, cmake
 , llvmPackages
-, pkg-config
+, libffi
+, libxml2
+, CoreFoundation
+, SystemConfiguration
+, Security
+, withLLVM ? (stdenv.isLinux || stdenv.isDarwin) && stdenv.isx86_64
+, withSinglepass ? (stdenv.isLinux || stdenv.isDarwin) && stdenv.isx86_64
```

Comment:
Not sure if we need the linux and darwin guards here because rust works on these platforms only anyway.

Reply:
Further relaxed the guard according to https://docs.wasmer.io/ecosystem/wasmer/wasmer-features#compiler-support-by-chipset